src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -877,6 +900,10 @@ void Pathfinder::addLink (
     auto const& uEndAccount = pathEnd.getAccountID ();
     bool const bOnXRP = uEndCurrency.isZero ();
 
+    // Does pathfinding really need to get this to
```

Comment:
This comment is a bit hard to parse (English-wise).  If you had a long comment for `mEffectiveDstAccount` you probably wouldn't need this comment at all...